lib/index.js

```diff
@@ -526,7 +519,7 @@ exports.updateRoles = brCallbackify(async (
  *
  * @returns {Promise} Resolves to an `actor` once the operation completes.
  */
-exports.getCapabilities = brCallbackify(async ({id, identities}) => {
+exports.getCapabilities = async ({id, identities} = {}) => {
```

Comment:
So `identities` should be removed here -- as well as any use of it within this function. This includes doing queries on the mongodb `identity` collection.

Reply:
There is also a mongodb index added to the `identity` collection up at the top in the `bedrock-mongodb.ready` function -- all references to the `identity` collection should be removed there as well.